storage-engine/titan-configuration.md

```diff
@@ -146,6 +162,10 @@ To fully disable Titan for all existing and future data, you can follow these st
     enabled = false
     ```
 
+### Data convertion speed from Titan to RocksDB
+
+Because Blob cache only helps when a value is accessed more than once, in compaction scenario, it's likely not useful. As a result, the data convertion from Titan to RocksDB can be 10x slower than RocksDB to Titan. In our test, a 800 GiB TiKV takes 12 hour to completely convert its data to RocksDB. 
```

Comment:
```suggestion
Because the values in Titan Blob files are not contiguous, and Titan's cache is at the value level, the Blob Cache does not help during compaction. The speed from Titan to RocksDB is an order of magnitude slower than the speed from RocksDB to Titan. In the test, it takes 12 hours for a volume of 800 GiB Titan data on a TiKV node to be converted to RocksDB by tikv-ctl to do full compaction.
```